src/gui/ogr/qgsnewogrconnection.cpp

```diff
@@ -138,4 +139,9 @@ void QgsNewOgrConnection::on_btnConnect_clicked()
   testConnection();
 }
 
+void QgsNewOgrConnection::helpRequest()
+{
+  QgsHelp::openHelp( "managing_data_source/opening_data.html#connecting-to-ogr" );
```

Comment:
Ok. then we can either use:
* http://docs.qgis.org/2.18/en/docs/user_manual/managing_data_source/opening_data.html#loading-a-layer-from-a-file which describes the path to access to that dialog (still in PR2000)
* or http://docs.qgis.org/2.18/en/docs/user_manual/managing_data_source/opening_data.html#creating-a-stored-connection which already describes the parameters in that dialog (might need to slightly add some cross references).